data/presets/leisure/bathing_place.json

```diff
@@ -0,0 +1,45 @@
+{
+    "aliases": [
+        "Bathing spot",
+        "Dip spot",
+        "Dipping spot",
+        "Swim spot",
+        "Swimming spot",
+        "Wild swimming spot"
+    ],
+    "icon": "fas-water-ladder",
+    "fields": [
+        "name",
+        "access_simple",
```

Comment:
What are the wikis recommendations or conventions on how to tag "this is an unofficial vs. official (signed) place"? Is it access or supervised? Maybe move supervised up?

Reply:
Good spot. That would be `informal` which I forgot to add to the preset.